### DISCUSSION TITLE
fa/manager/domain-worker-cdn-and-tunneling/Tutorial-for-setting-up-WST-tunnel-as-relay-server/

### DISCUSSION THREAD

**Discussion by giscus** [hiddify/Hiddify-Manager#4123] [Content Help کمک به اصلاح محتوا]
# fa/manager/domain-worker-cdn-and-tunneling/Tutorial-for-setting-up-WST-tunnel-as-relay-server/

The proxy manager platform.

https://hiddify.com/fa/manager/domain-worker-cdn-and-tunneling/Tutorial-for-setting-up-WST-tunnel-as-relay-server/

**Comment by Mammad3861** [comment]
مشکل داره؟ قبلا خوب کار میکرد اوبونتو 22.04
sudo wstunnel --server ws://0.0.0.0:8080 -v
/usr/bin/wstunnel: 1: Syntax error: "(" unexpected

**Reply by ReturnFI** [reply]
از اپ ریلی استفاده کنید
https://hiddify.com/fa/manager/domain-worker-cdn-and-tunneling/How-to-setup-relay-server-using-Hiddify-Relay-Builder/

**Reply by Mammad3861** [reply]
دوست عزیز روی اون هم این نوع تانل جواب نمیده
روش های تانل دیگه باعث مسدود شدن میشه طبق تجربم تنها این روش اوکی بوده برام

**Reply by ReturnFI** [reply]
روش Dokodemo-Door و HA-Proxy مدت ها هست در حال استفاده هستن و موردی نداشتن
در صورتی که خواستید میتونید با استفاده از این [آموزش ](https://hiddify.com/fa/manager/domain-worker-cdn-and-tunneling/Tutorial-for-setting-up-local-IPv6/) تانل IPv6 لوکال و Dokodemo استفاده کنید
